src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -0,0 +1,132 @@
+package uk.gov.companieshouse.service;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.when;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.companieshouse.exception.AppealNotFoundException;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.model.CreatedBy;
+import uk.gov.companieshouse.model.OtherReason;
+import uk.gov.companieshouse.model.PenaltyIdentifier;
+import uk.gov.companieshouse.model.Reason;
+import uk.gov.companieshouse.repository.AppealRepository;
+
+import java.util.Optional;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AppealServiceTest {
```

Comment:
I do not see any test for `saveAppeal` that would actually asses that creation time and user ID is associated with appeal prior save. These two things seems quite important. Argument captor could be used to achieve it.

Reply:
i have updated the `testCreateAppeal_returnsResourceId()` test to include argument captor